### ISSUE TITLE
Synfig does not add audio layers to MPG file when rendering.

### ISSUE THREAD

**Issue by CoryAlbrecht** [synfig/synfig#384]
Synfig 1.0.2 on Ubuntu GNOME 16.04.2

Synfig does not add audio layers to rendered MPG file. From the console output it looks like only a video stream is passed to ffmpeg but no audio stream.

```
cory@movingpictures[13:28:49]~/Documents/Synfig Projects/QGIS Backgrounder 1$ synfig --fps 25  -t ffmpeg -o qb1.mpg QGIS\ Backgrounder\ 1.sif 
synfig(31191) [01:30:01 PM] info: Loading modules from /etc/synfig/synfig_modules.cfg
/usr/bin/synfig: warning: Unable to find module "mod_libavcodec" (file not found)
synfig(31191) [01:30:01 PM] info: Loading file: QGIS Backgrounder 1.sif
synfig(31191) [01:30:01 PM] info: Render broken up into 1 block 781 pixels tall, and a final block 299 pixels tall
ffmpeg version 3.2-1001 Copyright (c) 2000-2016 the FFmpeg developers
  built with gcc 5.4.0 (Ubuntu 5.4.0-6ubuntu1~16.04.4) 20160609
  configuration: --prefix=/usr --extra-version=1001 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --enable-gpl --disable-libtesseract --disable-stripping --disable-decoder=libschroedinger --enable-avresample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libebur128 --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libmodplug --enable-libmp3lame --enable-libopenjpeg --enable-libopus --enable-libpulse --enable-librubberband --enable-libschroedinger --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-opengl --enable-sdl2 --enable-x11grab --enable-libdc1394 --enable-libiec61883 --enable-openal --enable-frei0r --enable-libopencv --enable-libx264 --enable-chromaprint --enable-shared
  libavutil      55. 34.100 / 55. 34.100
  libavcodec     57. 64.100 / 57. 64.100
  libavformat    57. 56.100 / 57. 56.100
  libavdevice    57.  1.100 / 57.  1.100
  libavfilter     6. 65.100 /  6. 65.100
  libavresample   3.  1.  0 /  3.  1.  0
  libswscale      4.  2.100 /  4.  2.100
  libswresample   2.  3.100 /  2.  3.100
  libpostproc    54.  1.100 / 54.  1.100
synfig(31191) [01:30:01 PM] warning: layers/layer_pastecanvas.cpp:471 bounding box shrank while rendering?
QGIS Backgrounder 1.sif ==> qb1.mpg: Frame 2 of 951 (0%). Remaining time: 3m 15s synfig(31191) [01:30:02 PM] info: Render broken up into 1 block 781 pixels tall, and a final block 299 pixels tall
[image2pipe @ 0x55631b908420] Stream #0: not enough frames to estimate rate; consider increasing probesize
Input #0, image2pipe, from 'pipe:':
  Duration: N/A, bitrate: N/A
    Stream #0:0: Video: ppm, rgb24, 1920x1080, 25 tbr, 25 tbn, 25 tbc
[mpeg @ 0x55631b912d20] VBV buffer size not set, using default size of 130KB
If you want the mpeg file to be compliant to some specification
Like DVD, VCD or others, make sure you set the correct buffer size
Output #0, mpeg, to 'qb1.mpg':
  Metadata:
    title           : "QGIS Backgrounder 1"
    encoder         : Lavf57.56.100
    Stream #0:0: Video: mpeg1video, yuv420p, 1920x1080, q=2-31, 200 kb/s, 25 fps, 90k tbn, 25 tbc
    Metadata:
      encoder         : Lavc57.64.100 mpeg1video
    Side data:
      cpb: bitrate max/min/avg: 0/0/200000 buffer size: 0 vbv_delay: -1
Stream mapping:
  Stream #0:0 -> #0:0 (ppm (native) -> mpeg1video (native))
synfig(31191) [01:30:02 PM] warning: layers/layer_pastecanvas.cpp:471 bounding box shrank while rendering?
QGIS Backgrounder 1.sif ==> qb1.mpg: Frame 3 of 951 (0%). Remaining time: 4m 24s synfig(31191) [01:30:02 PM] info: Render broken up into 1 block 781 pixels tall, and a final block 299 pixels tall
synfig(31191) [01:30:02 PM] warning: layers/layer_pastecanvas.cpp:471 bounding box shrank while rendering?
synfig(31191) [01:30:02 PM] warning: layers/layer_pastecanvas.cpp:471 bounding box shrank while rendering?
QGIS Backgrounder 1.sif ==> qb1.mpg: Frame 4 of 951 (0%). Remaining time: 5m 1s  synfig(31191) [01:30:03 PM] info: Render broken up into 1 block 781 pixels tall, and a final block 299 pixels tall
```

**Comment by rodolforg** [comment]
@morevnaproject  :)

**Comment by ice0** [comment]
Added a fix for MPEG-1 output (https://github.com/synfig/synfig/pull/1830). Now should works fine.

**Comment by rodolforg** [comment]
This issue is already fixed in v1.4.1.
